sql-to-dbsp-compiler/SQL-compiler/pom.xml

```diff
@@ -32,6 +32,18 @@
     <build>
         <finalName>sql2dbsp</finalName>
         <plugins>
+            <plugin>
```

Comment:
does this help?

Reply:
Build is less than around 1m30 on my machine when its cached.. if I only build
the compiler 40sec.. used to be 3-4min